doc/users/whats_new.rst

```diff
@@ -460,14 +497,71 @@ Example
     subplot2grid(shape, loc, rowspan=1, colspan=1, fig=myfig)
 
 
-Interpolation in fill_betweenx
-------------------------------
+Interpolation in ``fill_betweenx``
+----------------------------------
 
 The ``interpolate`` parameter now exists for the method
-:func:`~matplotlib.axes.Axes.fill_betweenx`. This allows a user to interpolate
-the data and fill the areas in the crossover points, similarly to
-:func:`~matplotlib.axes.Axes.fill_between`.
+:func:`~matplotlib.axes.Axes.fill_betweenx`. This allows a user to
+interpolate the data and fill the areas in the crossover points,
+similarly to :func:`~matplotlib.axes.Axes.fill_between`.
+
+
+New keyword argument ``sep`` for EngFormatter
+---------------------------------------------
+
+A new ``sep`` keyword argument has been added to
+:class:`~matplotlib.ticker.EngFormatter` and provides a means to
+define the string that will be used between the value and its
+unit. The default string is " ", which preserves the former
+behavior. Additionally, the separator is now present between the value
+and its unit even in the absence of SI prefix.  There was formerly a
+bug that was causing strings like "3.14V" to be returned instead of
+the expected "3.14 V" (with the default behavior).
+
+Extend ``MATPLOTLIBRC`` behavior
+--------------------------------
+
+The environmental variable can now specify the full file path or path
+to find :file:`matplotlibrc` in.
+
+
+``density`` kwarg to hist
+-------------------------
+
+The :meth:`~matplotlib.axes.Axes.hist` method now prefers ``density``
+to ``normed`` to control if the histogram should be normalized,
+following a change upstream to numpy.  This will reduce confusion as
+the behavior has always been that the integral of the histogram is 1
+(rather than sum or maximum value).
+
+
+
+Internals
++++++++++
+
+New TransformedPatchPath caching object
+---------------------------------------
+
+A newly added :class:`~matplotlib.transforms.TransformedPatchPath` provides a
+means to transform a :class:`~matplotlib.patches.Patch` into a
+:class:`~matplotlib.path.Path` via a :class:`~matplotlib.transforms.Transform`
+while caching the resulting path. If neither the patch nor the transform have
+changed, a cached copy of the path is returned.
+
+This class differs from the older
+:class:`~matplotlib.transforms.TransformedPath` in that it is able to refresh
+itself based on the underlying patch while the older class uses an immutable
+path.
+
+
+Abstract base class for movie writers
+-------------------------------------
 
+The new :class:`~matplotlib.animation.AbstractMovieWriter` class defines
+the API required by a class that is to be used as the ``writer`` in the
+:meth:`matplotlib.animation.Animation.save` method. The existing
+:class:`~matplotlib.animation.MovieWriter` class now derives from the new
+abstract base class.
 
 Validation of line style rcParams
 ---------------------------------
```

Comment:
just below, at "strings like ...": would add regular quotes (within the bq's) around "dashed" and "--"